packages/framework/aqueduct/package.json

```diff
@@ -141,6 +141,7 @@
 	"typeValidation": {
 		"broken": {
 			"InterfaceDeclaration_IDataObjectProps": {
+				"forwardCompat": false,
```

Comment:
Revert since we don't have changes in this package.

Reply:
Seems like it does affected this package. Type error was thrown after rebasing from main

Reply:
Oh, because the interface has a property of a type that we're breaking... so back to my general comment on the last review, I thought we had decided to make all changes here non-breaking?